tests/D2L.CodeStyle.Analyzers.Test/SpecRunner.cs

```diff
@@ -17,8 +8,10 @@
 using NUnit.Framework;
 
 namespace D2L.CodeStyle.Analyzers {
-	[TestFixtureSource( nameof( m_specNames ) )]
+
+	[TestFixtureSource( typeof( SpecTestsProvider ), nameof( SpecTestsProvider.GetAll ) )]

```

Comment:
Pulling all the inputs that were stored in statics out of fixture into test assembly specific `SpecTestsProvider`:

https://github.com/Brightspace/D2L.CodeStyle/pull/847/files#diff-200403da8446b3133b2a790dac3a198033138dae29d25a1151e707564f0a4435R10